lib/output.js

```diff
@@ -187,12 +187,15 @@ function OutputStream(options) {
             }
             might_need_space = false;
         }
-        var a = str.split(/\r?\n/), n = a.length - 1;
-        current_line += n;
-        if (n == 0) {
-            current_col += a[n].length;
-        } else {
-            current_col = a[n].length;
+        var a = str.split(/\r?\n/), len = a.length;
+        if (len > 0) {
+            var n = len - 1;
+            current_line += n;
+            if (n == 0) {
+                current_col += a[n].length;
+            } else {
+                current_col = a[n].length;
+            }
         }
```

Comment:
Oh, it's an old browser bug: `"aaaa".split(/a/)` should return an array with 5 empty strings, but in IE8 it returns an empty array.  Screw it, since you are using a polyfill anyway, [include this one](http://blog.stevenlevithan.com/archives/cross-browser-split) as well.  People who still use IE8 must suffer increased download times.